pkg/virtualKubelet/const.go

```diff
@@ -14,14 +14,16 @@
 
 package virtualKubelet
 
-import discoveryv1alpha1 "github.com/liqotech/liqo/apis/discovery/v1alpha1"
+import (
+	discoveryv1alpha1 "github.com/liqotech/liqo/apis/discovery/v1alpha1"
+)
 
 const (
 	// VirtualNodePrefix -> the prefix used to generate the virtual node name.
 	VirtualNodePrefix = "liqo-"
 )
 
-// VirtualNodeName generates the virtual node name based on the cluster ID.
-func VirtualNodeName(cluster *discoveryv1alpha1.ClusterIdentity) string {
+// VirtualNodeGroupName generate the group name for the virtual nodes referring a specific clusterID.
+func VirtualNodeGroupName(cluster *discoveryv1alpha1.ClusterIdentity) string {
```

Comment:
What does it mean "group"?

Reply:
We don't need it anymore. It was linked to the possibility to have nodes with the same name in different tenant.